.travis.yml

```diff
@@ -1,14 +1,44 @@
 sudo: required
```

Comment:
We also need to figure out how to run our message check script and report back. I imagine that we'll have to do some type of success code in the message check script (usually exit 0) and have travis ci pass if our script exits with success and fail if it doesn't